### PR TITLE
feat(tasks): Group tasks by status

### PR DESCRIPTION
**Pull request by nidzix**
SD-1173

### CODE REVIEW

File: app/scripts/superdesk-dashboard/workspace-tasks/tasks.js [hunk 1]
```diff
@@ -5,6 +5,12 @@
 TasksService.$inject = ['desks', '$rootScope', 'api'];
 function TasksService(desks, $rootScope, api) {
 
+    this.statuses =  [
```
Comment (amagdas) [inline app/scripts/superdesk-dashboard/workspace-tasks/tasks.js]: I think we should take this statuses from server...

**Comment by amagdas** [comment]
Besides the statuses, :+1:

**Comment by petrjasek** [comment]
:+1:

File: app/scripts/superdesk-dashboard/workspace-tasks/tasks.js (hunk 1 above)
Comment (nidzix) [inline app/scripts/superdesk-dashboard/workspace-tasks/tasks.js]: we can add this in another task?

Comment (amagdas) [inline app/scripts/superdesk-dashboard/workspace-tasks/tasks.js]: Yup, seems reasonable :+1:

**Comment by eozan** [comment]
:+1:

**Pull request merged**